src/main/resources/templates/mail/invitation_approval_execute_submission_text.txt

```diff
@@ -0,0 +1,8 @@
+Dear [(${userFirstName})],
+
+You have been asked to execute a submission in analysis [(@{${analysisUrl}})] for study [(${studyUrl})].
+To accept this request, please follow this link [(@{${invitationUrl}(id=${submissionId},accepted='true',type='APPROVE_EXECUTE_SUBMISSION',userUuid=${userUuid},token=${token})})].
+Alternatively, please follow this link [(@{${invitationUrl}(id=${submissionId},accepted='false',type='APPROVE_EXECUTE_SUBMISSION',userUuid=${userUuid},token=${token})})] if you wish to reject this request.
+
+Regards,
```

Comment:
The first example of redundancy issues: the signature is hardcoded, whereas it is configurable in the html templates